specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -410,6 +410,46 @@
         "deprecated": false
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.IoTCentral/appTemplates": {
```

Comment:
Still discussing about making it regional or global

Reply:
let me know when you have a decision.

Reply:
Right now, it's modeled as a top-level subscription resource. Such resources have to be ProxyOnly. ARM doesn't provide regional routing for top-level ProxyOnly resources, which will force them to be global.

If this is undecided, should we hold off on reviewing this PR?

---
In reply to: [270201167](https://github.com/Azure/azure-rest-api-specs/pull/5495#discussion_r270201167) [](ancestors = 270201167)

Reply:
@sergey-shandar @majastrz , we have decided to go with this global format

Reply:
Makes sense.

---
In reply to: [271399838](https://github.com/Azure/azure-rest-api-specs/pull/5495#discussion_r271399838) [](ancestors = 271399838)

Reply:
> let me know when you have a decision. #Closed

we just go with this format